tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,150 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations by the cross-cluster federation, unlimited storage, and global querying across clusters.
+
+Thanos builds upon vanilla Prometheus instances which states it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server in the eu1 region.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+mkdir test
+```{{execute}}
+
+```
+docker run -i dockerenginesonia/thanosbench:v1.4 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -i dockerenginesonia/thanosbench:v1.4 block gen --output.dir test
+```{{execute}}
+
+## Prometheus Configuration Files
+
+Here, we will prepare configuration files for the Prometheus instance.
+
+Click `Copy To Editor` for config to propagate the configs to file.
+
+<pre class="file" data-filename="prometheus0_eu1.yml" data-target="replace">
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+  external_labels:
+    cluster: eu1
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['127.0.0.1:9090']
+  - job_name: 'sidecar'
+    static_configs:
+      - targets: ['127.0.0.1:19090']
+  - job_name: 'minio'
+    metrics_path: /minio/prometheus/metrics
+    static_configs:
+      - targets: ['127.0.0.1:9000']
+  - job_name: 'store_gateway'
+    static_configs:
+      - targets: ['127.0.0.1:19090']
+</pre>
+
+## Starting Prometheus Instances
+
+Let's now start the container representing Prometheus instance.
+
+Execute the following commands:
+
+### Prepare "persistent volumes"
+
+```
+mkdir -p prometheus0_eu1_data
+```{{execute}}
+
+### Deploying "EU1"
+
+```
+docker run -d --net=host --rm \
+    -v $(pwd)/prometheus0_eu1.yml:/etc/prometheus/prometheus.yml \
+    -v $(pwd)/prometheus0_eu1_data:/prometheus \
```

Comment:
this one doesn't use the test directory u generated blocks in, is that on purpose?